lahja/endpoint.py

```diff
@@ -129,6 +175,65 @@ def connect_no_wait(self, loop: Optional[asyncio.AbstractEventLoop] = None) -> N
 
             self._process_item(item, config)
 
+    def _create_external_api(self, ipc_path: pathlib.Path) -> None:
+
+        receiver = EndpointConnector(self)
+
+        ConnectorManager = get_connector_manager()
```

Comment:
What do you think of making the flow a little simpler by in-lining this method? By in-lining it some code is duplicated but I think it makes the code easier to read. At first glance `get_connector_manager` looks like it returns a global instance, but looking at the definition reveals it's creating a fresh class each time.

Reply:
Yep, I was questioning the standalone function as well. Either inlining or just a private method on the endpoint itself. Which would you prefer?

Reply:
I think that I'd prefer in-lining, the docs give me the impression that creating the class and calling `.register` is a single operation, it'd be nice to have them be next to each other.

Reply:
Sounds good!